tests/unit/controllers/index-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'website-www/tests/helpers';
+
+module('Unit | Controller | index', function (hooks) {
+  setupTest(hooks);
+
+  // TODO: Replace this with your real tests.
```

Comment:
Remove it if not necessary

Reply:
It's a boilerplate code which we get from`ember generate` command, keeping as it is now